### PR TITLE
Unit 12. Generative models - Practical issues

### PR DESCRIPTION
**Pull request by hwaseem04**
Brief idea into the Ethical Issues associated with generative contents.

### CODE REVIEW

File: chapters/en/Unit 5 - Generative Models/PRACTICAL APPLICATIONS & CHALLENGES/Ethical Issues.mdx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+- **Transparency and labeling**: Platforms and developers are encouraged to be transparent about the use of AI-edited images and implement labeling systems to differentiate real and manipulated content.
+- **Fact-checking and verification**: Media outlets and tech companies are investing in fact-checking and verification tools to help combat the spread of misinformation and disinformation.
+- **Legal frameworks**: Governments are considering legislative measures to regulate the use of AI-edited images and hold individuals accountable for their misuse.
+
+
+## Future Scope
+The future of AI-edited images will likely involve:
+
+- **Advanced detection and mitigation techniques**: Researchers will ideally develop more advanced techniques for detecting and mitigating the harms associated with AI-edited images. But is like a cat and dog game, where one group develop sophisticated realistic AI images generating algorithms whereas another group develop methods to identify them
```
Comment (ricard-inho) [inline chapters/en/Unit 5 - Generative Models/PRACTICAL APPLICATIONS & CHALLENGES/Ethical Issues.mdx:31]:
```suggestion
- **Advanced detection and mitigation techniques**: Researchers will ideally develop more advanced techniques for detecting and mitigating the harms associated with AI-edited images. But is like a cat and dog game, where one group develops sophisticated realistic AI images generating algorithms whereas another group develops methods to identify them
```

**Review by ricard-inho — COMMENTED** [review]
I think something to consider if there's time is this [article](https://www.technologyreview.com/2023/10/23/1082189/data-poisoning-artists-fight-generative-ai/). It comes from the following [text](https://arxiv.org/abs/2310.13828). It talks about scraping copyright data for generative models.

File: chapters/en/Unit 5 - Generative Models/PRACTICAL APPLICATIONS & CHALLENGES/Ethical Issues.mdx [hunk 2]
```diff
@@ -0,0 +1,35 @@
+# Privacy, Bias and Societal Concerns
+
+The widespread adoption of AI-powered image editing tools raises significant concerns regarding privacy, bias, and potential societal ramifications. These tools, capable of manipulating both 2D and 3D images with remarkable realism, introduce ethical dilemmas and require careful consideration.
+
```
Comment (merveenoyan) [inline chapters/en/Unit 5 - Generative Models/PRACTICAL APPLICATIONS & CHALLENGES/Ethical Issues.mdx:4]:
```suggestion
```

File: chapters/en/Unit 5 - Generative Models/PRACTICAL APPLICATIONS & CHALLENGES/Ethical Issues.mdx [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+What you will learn from this chapter:
+
+    - Impact of such AI images/videos on Society
```
Comment (merveenoyan) [inline chapters/en/Unit 5 - Generative Models/PRACTICAL APPLICATIONS & CHALLENGES/Ethical Issues.mdx:8]:
```suggestion
- Impact of such AI images/videos on society
```
You can remove the indentations and capitals mid-sentence

File: chapters/en/Unit 5 - Generative Models/PRACTICAL APPLICATIONS & CHALLENGES/Ethical Issues.mdx [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+    - Current Approaches to tackle the issues
+    - Future Scope
+
+
+## Impact on Society
+The ability to effortlessly edit and alter images has the potential to:
+
+- **Undermine trust in media**: Deepfakes, convincingly manipulated videos, can spread misinformation and erode public trust in news and online content.
```
Comment (merveenoyan) [inline chapters/en/Unit 5 - Generative Models/PRACTICAL APPLICATIONS & CHALLENGES/Ethical Issues.mdx:16]:
```suggestion
- **Undermine trust in media:** Deepfakes, convincingly manipulated videos, can spread misinformation and erode public trust in news and online content.
```

File: chapters/en/Unit 5 - Generative Models/PRACTICAL APPLICATIONS & CHALLENGES/Ethical Issues.mdx (hunk 1 above)
Comment (merveenoyan) [inline chapters/en/Unit 5 - Generative Models/PRACTICAL APPLICATIONS & CHALLENGES/Ethical Issues.mdx:31]:
```suggestion
- **Advanced detection and mitigation techniques**: Researchers will ideally develop more advanced techniques for detecting and mitigating the harms associated with AI-edited images. But it is like a cat and dog game: where one group develops sophisticated, realistic image generation algorithms, whereas another group develops methods to identify them.
```

File: chapters/en/Unit 5 - Generative Models/PRACTICAL APPLICATIONS & CHALLENGES/Ethical Issues.mdx [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+- **Transparency and labeling**: Platforms and developers are encouraged to be transparent about the use of AI-edited images and implement labeling systems to differentiate real and manipulated content.
+- **Fact-checking and verification**: Media outlets and tech companies are investing in fact-checking and verification tools to help combat the spread of misinformation and disinformation.
+- **Legal frameworks**: Governments are considering legislative measures to regulate the use of AI-edited images and hold individuals accountable for their misuse.
+
+
+## Future Scope
+The future of AI-edited images will likely involve:
+
+- **Advanced detection and mitigation techniques**: Researchers will ideally develop more advanced techniques for detecting and mitigating the harms associated with AI-edited images. But is like a cat and dog game, where one group develop sophisticated realistic AI images generating algorithms whereas another group develop methods to identify them
+- **Public awareness and education**: Public awareness campaigns and educational initiatives will be crucial in promoting responsible use of AI-edited images and combating the spread of misinformation.
```
Comment (merveenoyan) [inline chapters/en/Unit 5 - Generative Models/PRACTICAL APPLICATIONS & CHALLENGES/Ethical Issues.mdx:32]:
```suggestion
- **Public awareness and education:** Public awareness campaigns and educational initiatives will be crucial in promoting responsible use of AI-edited images and combating the spread of misinformation.
```

**Review by merveenoyan — COMMENTED** [review]
Thanks a lot for this section! Can you make the file names consistent, i.e.  chapters/en/Unit 5 - Generative Models/Practical Applications and Challenges/Ethical Issues.mdx

**Review by WilliamBonvini — COMMENTED** [review]
Why there are all those empty README.md files? Maybe they don't cause any problem, but I don't think they are supposed to be there, right?

File: chapters/en/Unit 5 - Generative Models/PRACTICAL APPLICATIONS & CHALLENGES/Ethical Issues.mdx [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+
+
+This is a rapidly evolving field, and it is crucial to stay informed about the latest developments.
```
Comment (johko) [inline chapters/en/Unit 5 - Generative Models/PRACTICAL APPLICATIONS & CHALLENGES/Ethical Issues.mdx:35]: Do you have any sources for the ethical part people can follow - websites, social media accounts? Maybe would be nice if you know any. But if not, that is okay as well :)

**Review by johko — APPROVED** [review]
LGTM :+1:

File: chapters/en/Unit 5 - Generative Models/PRACTICAL APPLICATIONS & CHALLENGES/Ethical Issues.mdx [hunk 7]
```diff
@@ -0,0 +1,35 @@
+# Privacy, Bias and Societal Concerns
+
+The widespread adoption of AI-powered image editing tools raises significant concerns regarding privacy, bias, and potential societal ramifications. These tools, capable of manipulating both 2D and 3D images with remarkable realism, introduce ethical dilemmas and require careful consideration.
+
+
+What you will learn from this chapter:
+
+    - Impact of such AI images/videos on Society
+    - Current Approaches to tackle the issues
+    - Future Scope
```
Comment (WilliamBonvini) [inline chapters/en/Unit 5 - Generative Models/PRACTICAL APPLICATIONS & CHALLENGES/Ethical Issues.mdx:10]: I would replace the capital letters also here ("Approaches" --> "approaches" and "Scope" --> "scope")

File: chapters/en/Unit 5 - Generative Models/PRACTICAL APPLICATIONS & CHALLENGES/Ethical Issues.mdx (hunk 1 above)
Comment (WilliamBonvini) [reply chapters/en/Unit 5 - Generative Models/PRACTICAL APPLICATIONS & CHALLENGES/Ethical Issues.mdx:31]: "But is like a cat and dog game" --> "But **it's** like a **cat-and-mouse** game"

**Review by WilliamBonvini — CHANGES_REQUESTED** [review]

**Comment by hwaseem04** [comment]
> Looking good so far! I think something to consider if there's time is this [article](https://www.technologyreview.com/2023/10/23/1082189/data-poisoning-artists-fight-generative-ai/). It comes from the following [text](https://arxiv.org/abs/2310.13828). It talks about scraping copyright data for generative models.

That is an interesting read. Probably will add it as a reference reading material

**Comment by hwaseem04** [comment]
> Why there are all those empty README.md files? Maybe they don't cause any problem, but I don't think they are supposed to be there, right?

Thank you for pointing out. Those empty files are removed.

File: chapters/en/Unit 5 - Generative Models/PRACTICAL APPLICATIONS & CHALLENGES/Ethical Issues.mdx (hunk 6 above)
Comment (hwaseem04) [reply chapters/en/Unit 5 - Generative Models/PRACTICAL APPLICATIONS & CHALLENGES/Ethical Issues.mdx:35]: That sounds good, will try to add some.

**Review by merveenoyan — APPROVED** [review]
LGTM!

**Pull request merged**